libs/core/src/components/pds-link/pds-link.tsx

```diff
@@ -12,6 +12,11 @@ import { launch } from '@pine-ds/icons/icons';
   shadow: true,
 })
 export class PdsLink {
+  /**
+   * Sets the link color.
+   */
+  @Prop() color?: 'secondary' | 'accent' | 'danger';
```

Comment:
Do we only want to restrict to these? I know in other components, we kind of opened it up.

Reply:
I thought about that, but I don't really see there being a need for someone to have a color outside of this for links. My thought was if we do, we can just refactor later.

Reply:
I would agree with @ju-Skinner here. Or we should at least match the `pds-text` colors, maybe? I could imagine links being inserted into text content.

We also have others, like mercury, in the app as well.

![Screenshot 2025-05-28 at 2 22 48 PM](https://github.com/user-attachments/assets/d1b46bad-e1ff-4192-bf02-81cbd306e05b)

Reply:
This has been updated.